lbjava/doc/ALGORITHMS.md

```diff
@@ -1 +1,94 @@
+            * [`SparseAveragedPerceptron`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseAveragedPerceptron.java)
+        * [`SparseConfidenceWeighted`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseConfidenceWeighted.java)
+        * [`SparseWinnow`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseWinnow.java)
+    * [`AdaBoost`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/AdaBoost.java)
+    * [`AdaGrad`]()
+    * [`MuxLearner`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/MuxLearner.java)
+    * [`NaiveBayes`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/NaiveBayes.java)
+    * [`SparseMIRA`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseMIRA.java)
+    * [`SupportVectorMachine`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SupportVectorMachine.java)
+    * [`SparseNetworkLearner`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseNetworkLearner.java)    
+        * [`MultiLabelLearner`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/MultiLabelLearner.java)
+        
+### Note on Binary & Multiclass Classification
+
+##### Please use [`SparseNetworkLearner`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseNetworkLearner.java) for both binary and multiclass classification.
+
+##### Please avoid using learning algorithms, such as [`SparseWinnow`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseWinnow.java), [`SparsePerceptron`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparsePerceptron.java), and [`SparseAveragedPerceptron`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseAveragedPerceptron.java) directly.
+
+The code snippets below demonstrated how to use learning algorithms inside [`SparseNetworkLearner`](https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/learn/SparseNetworkLearner.java) programmatically, and how to set parameters accordingly.
+
+#### Declarations in `.lbj` file, with only `SparseNetworkLearner`
+```
```

Comment:
Could you add `java` after each triple-quite? It would add java highlight to the code.